### ISSUE TITLE
RoslynPad Editor in Blazor

### ISSUE THREAD

**Issue by cordasfilip** [roslynpad/roslynpad#438]
Would be cool to enable RoslynPad editor in Blazor using the monaco editor component.
Should be possible to do for Blazor server and Blazor Hybrid(It runs regular core framework with no limitations) but might be possible to run in WASM haven't really investigated.
Just an idea could be fun to do.

**Comment by aelij** [comment]
Cool idea :) I wish I had time to tinker with this

**Comment by CoolCoderSuper** [comment]
Wouldn't it be possible to use Avalonia UI in the browser to achieve this?

**Comment by cordasfilip** [comment]
@CoolCoderSuper I think this question was before avalonia had web support. As I said full WASM component might not work. Didn't have time to play with this either and last time I tried avalonia wasn't working but could be me not trying enough. Still there are reasons to use Monaco editor it's pretty much bug free and more stable than avaloniaedit, supports accessibility...

**Comment by CoolCoderSuper** [comment]
I agree Monaco would likely be more stable and feature rich there is already Avalonia UI bindings so it might be easier with them.
With Avalonia v11 i have pretty good success running on web and mobile platforms.
However I'm unsure how well Roslyn would work depending on how it loads any helper assemblies.

**Comment by CoolCoderSuper** [comment]
SharpLab and mirrorsharp might be useful, I have used mirrosharp to build a script editor and worked pretty good. It's based on code mirror.
